<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="symbol_label">
         <property name="text">
          <string>Ticker Symbol</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="stock_symbol">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Exchange Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="exchange_name"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="stock_interval_label">
     <property name="text">
      <string>Stock Interval</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="daily_button">
       <property name="text">
        <string>Daily</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">stock_interval</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="weekly_button">
       <property name="text">
        <string>Weekly</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">stock_interval</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="intraday_button">
       <property name="text">
        <string>Intraday</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">stock_interval</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="day_interval_label">
     <property name="text">
      <string>Day Interval</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="one_min_button">
       <property name="text">
        <string>1</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">intraday_interval</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="five_min_button">
       <property name="text">
        <string>5</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">intraday_interval</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="fifteen_minute_button">
       <property name="text">
        <string>15</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">intraday_interval</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="thirty_minute_button">
       <property name="text">
        <string>30</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">intraday_interval</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="sixty_minute_button">
       <property name="text">
        <string>60</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">intraday_interval</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="data_amount_label">
     <property name="text">
      <string>Data Amount</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="output_full">
       <property name="text">
        <string>Full</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="output_compact">
       <property name="text">
        <string>Compact</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="one_more_thing">
     <property name="text">
      <string>one more thing</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QRadioButton" name="show_stock">
       <property name="text">
        <string>Show Stock Price</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="show_volume">
       <property name="text">
        <string>Show Stock Volume</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="go_button">
     <property name="text">
      <string>Go</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="plot_widget" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="stock_interval"/>
  <buttongroup name="intraday_interval"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
